libs/Microsoft.MixedReality.WebRTC/Interop/DeviceAudioTrackSourceInterop.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
nit: given that these are internal and not a lot of code, I'd keep them in the original file, just for the sake of not scrambling the line history.

Reply:
I'd rather keep things consistent so we can more easily find where things are, than randomly merge some classes into some files while keeping others in separate files. Again, I intended to have external audio sources as a second derived class from that same base `AudioTrackSource`, and cramming everything in a single file sounds confusing. Hopefully I can add that change soon, but waiting on guidance from Google.

Reply:
If we plan to put things in AudioTrackSourceInterop this should be fine then.

> I'd rather keep things consistent so we can more easily find where things are, than randomly merge some classes into some files while keeping others in separate files.

I see your point, though in general I believe symmetry in the file system is not that important (for me at least) when one browses the code mostly using search and jumping from point of use to definition.